model/patch/patch.go

```diff
@@ -51,6 +51,7 @@ type Patch struct {
 	PatchedConfig   string           `bson:"patched_config"`
 	Alias           string           `bson:"alias"`
 	GithubPatchData GithubPatch      `bson:"github_patch_data,omitempty"`
+	CommitQueue     bool             `bson:"commit_queue,omitempty"`
```

Comment:
A possible alternative to adding a field in order to determine if a patch is destined for the commit queue is to check the patch's Alias.

Reply:
Seems fine to have it on the patch document, the situations with the intents are a bit weird. If we had a working comit queue for github, why do we need this change now? Additionally, the requester on the version should tell us this, if needed.

Reply:
On the flip side, if we are saving a new field of whether it's a commit queue patch, it might make sense to save it for  GitHub patches as well, since it may be useful elsewhere to be able to check if a patch is for the commit queue.

Reply:
I'm not sure if the version requester will help us since the version won't have been created when we want to show the queue position.

Reply:
I definitely didn't see the Alias field. That's a much better alternative thank you!